receiver/cloudfoundryreceiver/converter.go

```diff
@@ -41,15 +47,34 @@ func convertEnvelopeToMetrics(envelope *loggregator_v2.Envelope, metricSlice pme
 	}
 }
 
+func convertEnvelopeToLogs(envelope *loggregator_v2.Envelope, logSlice plog.LogRecordSlice, startTime time.Time) error {
+	log := logSlice.AppendEmpty()
+	log.SetTimestamp(pcommon.Timestamp(envelope.GetTimestamp()))
+	log.SetObservedTimestamp(pcommon.NewTimestampFromTime(startTime))
+	logLine := string(envelope.GetLog().GetPayload())
+	log.Body().SetStr(logLine)
+	//exhaustive:enforce
+	switch envelope.GetLog().GetType() {
+	case loggregator_v2.Log_OUT:
+		log.SetSeverityText(plog.SeverityNumberInfo.String())
+		log.SetSeverityNumber(plog.SeverityNumberInfo)
+	case loggregator_v2.Log_ERR:
+		log.SetSeverityText(plog.SeverityNumberError.String())
+		log.SetSeverityNumber(plog.SeverityNumberError)
+	default:
+		return errors.New("unsupported log type")
```

Comment:
It would probably be helpful to include the value of `envelope.GetLog().GetType()` in the error to know what support is missing, or what the kind of error state we got into.